lib/pbench/cli/agent/commands/tools/list.py

```diff
@@ -13,6 +13,13 @@
 from pbench.cli.agent import CliContext, pass_cli_context
 from pbench.cli.agent.commands.tools.base import ToolCommand
 from pbench.cli.agent.options import common_options
+from pbench.agent.tool_group import BadToolGroup
+
+# FIX:
+# the set of error codes is currently
+# 0
+# 1 = Bad tool group
+# 2 = anything else
```

Comment:
Not sure this exit code is needed.

Reply:
There are situations other than a bad tool group that seem to require an error status - no?